src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
```

Comment:
The class name `RightTriangle` is appropriate as it represents a specific shape, but ensure that other classes and interfaces follow the convention of not using verbs for names.